.circleci/config.yml

```diff
@@ -135,11 +135,11 @@ jobs:
 
       - runtests:
           platform: jdk8
-          testjdk: -Dtest.java.home=/usr/lib/jvm/java-8-openjdk-amd64
+          testjdk: -Dtest.java.version=8
 
   testjdk8alpn:
     docker:
-      - image: circleci/openjdk:11.0.6-jdk-stretch
```

Comment:
I suspect I may need to use a custom path and revert this docker image.  Will follow up on this.

org.gradle.java.installations.paths=/custom/path/jdk1.8,/shared/jre11